scripts/exwave_stat.sh

```diff
@@ -455,26 +455,31 @@
     FH3=$(printf "%03d" $fhour)
     for stype in mean spread probab
     do
-     fcopy=${WAV_MOD_TAG}.${stype}.t${cyc}z.f${FH3}.grib2
-     if [ -s ${fcopy} ]
-     then
-       set +x
-       echo "   Copying ${fcopy} to $COMOUT/gridded"
-       [[ "$LOUD" = YES ]] && set -x
-       cp -f ${fcopy} $COMOUT/gridded
-      else
+      fcopy=${WAV_MOD_TAG}.${stype}.t${cyc}z.f${FH3}.grib2
+      if [ -s ${fcopy} ]
+      then
         set +x
-        echo ' '
-        echo '*************************************** '
-        echo "*** FATAL ERROR: No ${fcopy} file found *"
-        echo '*************************************** '
-        echo ' '
-        echo "$modIE fcst $date $cycle: ${fcopy} not fouund." >> $wavelog
-        echo $msg
+        echo "   Copying ${fcopy} to $COMOUT/gridded and ALERT if SENDDBN=YES"
         [[ "$LOUD" = YES ]] && set -x
-        export err=6;${errchk}
-        exit $err
-      fi
+        cp -f ${fcopy} $COMOUT/gridded
```

Comment:
Copy to COM does not live in a SENDCOM block

Reply:
Added SENDCOM dependency for copying grib2 file to exwave_stat.sh